chromium_src/components/content_settings/core/common/cookie_settings_base.cc

```diff
@@ -143,6 +143,8 @@ bool BraveIsAllowedThirdParty(
   return false;
 }
 
+bool IsThirdPartyRequest(const GURL& url, const GURL& site_for_cookies);
```

Comment:
I'm a bit confused by this. Is this the definition of the function?

Reply:
Ah, this is a forward-declaration of a static function that is in the Chromium version of `cookie_settings_base.cc`. Got it. Thanks!